neo/test/rawiotest/test_spikegadgetsrawio.py

```diff
@@ -16,6 +17,38 @@ class TestSpikeGadgetsRawIO(
         "spikegadgets/SpikeGadgets_test_data_2xNpix1.0_20240318_173658.rec",
     ]
 
+class TestSpikeGadgetsRawIOHeaderOnly(unittest.TestCase):
+    def setUp(self):
+        filename = "/Volumes/T7/CatalystNeuro/Jadhav/stubbed_files/SL18_D19_S01_F01_BOX_SLP_20230503_112642_stubbed.rec"
```

Comment:
something tells me this line won't work for the CI

Reply:
Yes, this will need to be updated once the test file is up on the gin repo.